src/main/java/com/google/ehub/data/CommentData.java

```diff
@@ -0,0 +1,14 @@
+package com.google.ehub.data;
+
+public class CommentData {
+  private final Long itemId;
+  private final String message;
+  private final Long timestampMillis;
+
+  public CommentData(long itemId,String message, long timestampMillis){
```

Comment:
There should be a space before the curly brace. Here and in other files too. 

```suggestion
  public CommentData(long itemId, String message, long timestampMillis) {
```